public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>Mac Calculator Clone</title>
+
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="./index.css">
+</head>
+<body>
+  <div id="calculator" class="calculator-container">
```

Comment:
I never use IDs and always use css and unique classnames. consider changing your classname `. calculator-container` to `.calculator` and then in your js use `document.querySelector('.calculator')`